source/config/configFlags.py

```diff
@@ -251,3 +251,30 @@ def _displayStringLabels(self):
 			# Translators: A label for an option to choose a method of reporting information, e.g. font attributes.
 			self.SPEECH_AND_BRAILLE: _("Speech and braille"),
 		}
+
+
+class ParagraphStartMarker(DisplayStringStrEnum):
+	NONE = ""
+	SPACE = " "
+	# Translators: This is a paragraph start marker used in braille.
+	# The default symbol is the pilcrow,
+	# a symbol also known as "paragraph symbol" or "paragraph marker".
+	# This symbol should translate in braille via LibLouis automatically.
+	# If there is a more appropriate character for your locale,
+	# consider overwriting this (e.g. the reversed pilcrow ⁋).
```

Comment:
Here's a real world example of why this is useful - since several languages have their own symbol for this

```suggestion
	# consider overwriting this (e.g. for Ge'ez ፨).
```

https://en.wikipedia.org/wiki/Pilcrow#Paragraph_signs_in_non-Latin_writing_systems